src/main/webapp/3Dscript.js

```diff
@@ -0,0 +1,197 @@
+import {FlyControls} from 'https://threejs.org/examples/jsm/controls/FlyControls.js';
+import {OrbitControls} from 'https://threejs.org/examples/jsm/controls/OrbitControls.js';
+import {GUI} from 'https://threejs.org/examples/jsm/libs/dat.gui.module.js';
+
+// These are global objects.
+let scene;
+let camera;
+let renderer;
+let clock;
+let controls;
+let orientation;
+let trajectory;
+let pose;
+let poseTransform = {
+  translateX: 0,
+  translateZ: 0,
+  rotate: 0,
+  scale: 0};
```

Comment:
Wouldn't 1 be a more sensible default for scale? I think 0 would collapse everything to a singularity.